client/src/track.ts

```diff
@@ -341,13 +341,28 @@ export default class Track {
     }
   }
 
-  getType(): [string, number] | null {
+  getType(index = 0): [string, number] | null {
     if (this.confidencePairs.length > 0) {
+      if (this.confidencePairs[index]) {
+        return this.confidencePairs[index];
+      }
```

Comment:
This doesn't seem right.

if confidencePairs is `[['foo', 0.5], ['bar', 0.6]]` and I call getType(20000), I'll get `['foo', 0.5]`.  I think I should get `null`.

Reply:
I was maintaining parity with the existing getType() which returned the 0 indexed confidencePair if it existed or else it would return `null`.  So I modified it to return the 0 index confidencePair by default if it existed and `index` was out of scope.  Or else it would return null if the confidencePairs were empty.

We can change this but I'll need to review and understand why we had it like this in the first place.  I imagine it has something to do with creation or CSV files which don't always contain types.